lib/dockerregistry/inventory_test.go

```diff
@@ -1315,6 +1315,34 @@ func TestReadRegistries(t *testing.T) {
 	}
 }
 
+func TestGetTokenKeyDomainRepoPath(t *testing.T) {
+	type TokenKeyDomainRepoPath [3]string
+	var tests = []struct {
+		name     string
+		input    RegistryName
+		expected TokenKeyDomainRepoPath
+	}{
+		{
+			"basic",
+			"gcr.io/foo/bar",
+			[3]string{"gcr.io/foo", "gcr.io", "foo/bar"},
+		},
+	}
+
+	for _, test := range tests {
+		tokenKey, domain, repoPath := GetTokenKeyDomainRepoPath(test.input)
```

Comment:
One thing I've found fairly recently is to call `t.Run(test.Name, func (t *testing.T) { ...` and then we don't need to log the name every time.

Reply:
ACK